formats/buttwoo.js

```diff
@@ -0,0 +1,48 @@
+const SSBURI = require('ssb-uri2')
+const butt2 = require('ssb-buttwoo')
```

Comment:
This import is not used? IMO it could be used, we don't need to pluck the feedFormat from a db2 API (and I think findFeedFormat is going to be an internal API)